client/web/src/nav/NavBar/NavDropdown.tsx

```diff
@@ -26,33 +33,33 @@ interface NavDropdownProps {
     items: NavDropdownItem[]
 }
 
-const DROPDOWN_MODIFIERS = {
-    flip: {
-        enabled: false,
-    },
-    offset: {
-        enabled: true,
-        // Offset menu to the top so that the menu overlaps with the toggle button.
-        // This prevents the menu from closing when moving mouse cursor from the button
-        // to the menu.
-        offset: '-10,-2',
-    },
-}
-
 export const NavDropdown: React.FunctionComponent<NavDropdownProps> = ({ toggleItem, mobileHomeItem, items }) => {
     const location = useLocation()
     const history = useHistory()
+    const menuListReference = useRef<HTMLDivElement | null>(null)
 
     const [isDropdownOpen, setIsDropdownOpen] = useState(false)
-    const toggle = (event: React.KeyboardEvent | React.MouseEvent): void => {
+    const handleKeyDown = (event: React.KeyboardEvent): void => {
         // Don't toggle dropdown with Enter key; instead, navigate to the home item of the dropdown.
         // This matches the behavior of the rest of the nav items.
-        if (event.type === 'keydown' && (event as React.KeyboardEvent).key === 'Enter') {
-            history.push(toggleItem.path)
-            return
+        if (event.type === 'keydown') {
+            switch (event.key) {
+                case 'Enter': {
+                    history.push(toggleItem.path)
+                    return
+                }
+                case 'ArrowDown': {
```

Comment:
```suggestion
                case 'Enter': {
                    history.push(toggleItem.path)
                    event.preventDefault();
                    return
                }
                case 'ArrowUp':
                case 'ArrowDown': {
```

This is the behavior we want. It'll match or normal Menu components and ensure that the dropdown doesn't open when the link is clicked

Reply:
Resolved